Fixed the storing point for the microagent's resource info

ssgUPC opened the pull request:
@cjdcordeiro the new ARM docker image for Res-Cat is `resCatlatest-V2.0.28-arm` which is in the `mf2c\trunk` repo and also I have modified the docker-image tag for microagent. Now the new version is - `1.0.5`. So, before you accept the PR, please build the microagent image with the new res-cat arm image and then accept it. I'd also like to inform you that @ALEJANDROJ19 has been already reviewed my code.

cjdcordeiro commented:
what is this fixing?

ALEJANDROJ19 commented:
> what is this fixing?

https://github.com/mF2C/ResourceManagement/issues/84
and I think that also this one: https://github.com/mF2C/ResourceManagement/issues/80

ALEJANDROJ19 commented:
@ssgUPC missing docs

ssgUPC commented:
@ALEJANDROJ19 I think, for `microagent`, we don't have any docs for each component. 

@cjdcordeiro,  @ALEJANDROJ19 has been already mentioned the ticket numbers. Just in a brief, I'd like to inform you, the problem I fixed with the new ARM res-cat image. Initially, it was always storing the microagent resource information to the cloud agent (which is running on the dashboard), but when a leader has been connected with the microagent, then it should store the resource info to the leader.  But unfortunately, it wasn't the case. So, I fixed this issue now when a microagent will be connected to the leader...it will store its resource info to the leader side.
